substrate/client/consensus/common/src/block_import.rs

```diff
@@ -43,7 +43,7 @@ pub enum ImportResult {
 }
 
 /// Auxiliary data associated with an imported block result.
-#[derive(Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
dq: Does this need the `Clone` here? Wouldn't the `ImportResult` and the rest also need it?

Reply:
The clone is needed because it's part of result of a RPC method call.

Thus, no `ImportResult` is not part of that and no Clone needed.